src/charm.py

```diff
@@ -374,6 +374,15 @@ def _on_workload_tracing_endpoint_removed(self, _) -> None:
 
     @property
     def _catalogue_item(self) -> CatalogueItem:
+        api_endpoints = {
+            "Instant query": "/loki/api/v1/query",
+            "Query range": "/loki/api/v1/query_range",
+            "Labels": "/loki/api/v1/labels",
+            "Label values": "/loki/api/v1/label/<name>/values",
+            "Prometheus alerts": "/prometheus/api/v1/alerts",
+            "Prometheus rules": "/prometheus/api/v1/alerts",
```

Comment:
Same comment as Loki coordinator, these are not Prometheus rules, but Loki rules; it's just a Prometheus-compatible endpoint.